node/flow.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  parser: 'babel-eslint',
+  extends: 'mixmax/node',
+  plugins: [
+    // Allows correct parsing of flow annotated files.
+    'flowtype'
+  ],
+  rules: {
+    'space-infix-ops': 0,
+    'no-duplicate-imports': 0,
+    // Makes flow `type` definitions defines
+    'flowtype/define-flow-type': 1,
+    // Do not allow spaces before type declarations, i.e. let a: number, not let a : number
+    'flowtype/space-before-type-colon': [
+      1,
+      'never'
+    ],
+    // Marks flow type declarations as used to prevent no-unused-vars for types
+    'flowtype/use-flow-type': 1,
+    // Enforces camelcase for type definitions
+    'flowtype/type-id-match': [
+      2,
+      '^([A-Z]+[a-z0-9A-Z]*)$'
+    ]
+  },
+  settings: {
+    // Only use the flow linter for files with `@flow`
+    flowtype: {
+      onlyFilesWithFlowAnnotation: true
+    }
+  },
+  parserOptions: {
+    // Required to support import/export syntax when using types
+    sourceType: 'module',
+    ecmaFeatures: {
+      experimentalObjectRestSpread: true,
```

Comment:
what do you think about breaking out a new config to be used by transpiled, node-run code and `extend`ing that config here? here and in my previous comment, just trying to think about how we can best keep these configs DRY - we might want several experimental features in multiple locations, but won't want to copy/paste those everywhere.

here, maybe we want to combine two different configs like `flow` and `node-es6` to create the final product you'd use in a flow-using, transpiled, server-side shared module.